### ISSUE TITLE
PG::Error: ERROR:  multiple assignments to same column "last_post_at" (ActiveRecord::StatementInvalid)

### ISSUE THREAD

**Issue by nebiros** [rubysherpas/forem#245]
I got this error when I run `rails g forem:install`, here's the error:

```
Creating default forum and topic
/Users/nebiros/.rbenv/versions/1.9.3-p194/lib/ruby/gems/1.9.1/gems/activerecord-3.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:1151:in `async_exec': PG::Error: ERROR:  multiple assignments to same column "last_post_at" (ActiveRecord::StatementInvalid)
: UPDATE "forem_topics" SET "last_post_at" = '2012-06-14 20:51:58.235960', "last_post_at" = '2012-06-14 20:51:58.235960', "updated_at" = '2012-06-14 20:51:58.278773' WHERE "forem_topics"."id" = 1
```

The full log: http://pastie.org/private/dj5ypqhtv0egd8r86vsyq

**Comment by radar** [comment]
This is now fixed with your commits. Thanks!

**Comment by radar** [comment]
Just FYI: Your fix introduced a bug (see #255). This was entirely my fault because I wrote the last_post_at test poorly. If I didn't do that then the test would've failed and you would've seen the mistake. 

I've now fixed this with @ad618f98b3fc2de86b500c56aca4be1ac99dae9e.

**Comment by nebiros** [comment]
@radar Ouch, sorry :)
